fictadvisor-api/src/v2/api/controllers/TeacherController.ts

```diff
@@ -508,4 +510,42 @@ export class TeacherController {
   ) {
     return this.teacherService.connectTeacherWithCathedra(teacherId, cathedraId);
   }
+
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: TeacherWithRolesAndContactsResponse,
```

Comment:
Add description properties in `TeacherWithRolesAndContactsResponse`
I meant that you have to describe every property of this class

![image](https://github.com/fictadvisor/fictadvisor/assets/114052215/52612e1d-cffb-493d-8f27-17540ebb0e31)